java-source/src/test/java/com/example/contract/IOUFlowTests.java

```diff
@@ -41,44 +44,34 @@ public void tearDown() {
         net.stopNodes();
     }
 
-    @Test
-    public void flowRejectsInvalidIOUs() throws InterruptedException {
+    @Test(expected=ExecutionException.class)
```

Comment:
Fair enough I can see that. In that case we need to be doing assertions on exceptions properly - either use what's available in JUnit or use something like assertj.